dation.css

```diff
@@ -0,0 +1,703 @@
+:root {
+    scroll-behavior: smooth;
+    /* --primary-color: white;
+    --secondary-color: black; */
+    
+    /* --primary-color: #1DA1F2;
+    --secondary-color: black; */
+    
+    /* --primary-color: #1DA1F2;
+    --secondary-color: white; */
+
+    /* --primary-color: white;
+    --secondary-color: black; */
+
+    --primary-color: black;
+    --secondary-color: white;
+    
+    --shadow-color: #9CA3AF;
+    /* --glow-color: blue; */
+    --glow-color: green;
+
+    --dark-grey: #6B7280;
+    --off-white: #F3F4F6;
+    
+    /* --nav-bg-color: #1DA1F2; */
+    /* --nav-bg-color: black; */
+    --nav-bg-color: var(--primary-color);
+    --nav-txt-color: var(--secondary-color);
+
+    --padding-xs: 0.3em;
+    --padding-s: 0.7em;
+    --padding-m: 1em;
+    --padding-l: 1.3em;
+    --padding-xl: 2em;
+
+    --margin-xs: 0.3em;
+    --margin-s: 0.7em;
+    --margin-m: 1em;
+    --margin-l: 1.3em;
+    --margin-xl: 2em;
```

Comment:
few CSS variables can be avoided.  instead of two separate spacing values for padding and margin could have used a single group for small, medium, large and extra large units and used those for padding and margin

Reply:
Good suggestion. Added 5 space variables for respective xs, s, m, l and xl.